src/components/bookmarks/addBookmark/addBookmark.js

```diff
@@ -216,13 +205,21 @@ class AddBookmark extends React.Component {
                       readOnly={fields[field.name].readonly}
                       size="l"
                       autoComplete="off"
+                      feedback={fields[field.name].feedback}
+                      {...{}/* eslint-disable-next-line no-nested-ternary */}
+                      status={fields[field.name].value && !fields[field.name].readonly
```

Comment:
I think will be better make this validation take place in a function so instead of have this in here, you can pass the values that you need for the validation to a function, like

```
validateStatus(fields) { ... logic }

...
status={this.validateStatus(fields)}
/>
```

Reply:
I managed to avoid the nested ternary without extra method by moving the logic inside Input, see https://github.com/LiskHQ/lisk-hub/pull/2411/commits/2bcd1d9ffa0029e7048ca29e4e9b214c2e7dae32 and let me know if you think it's enough